src/st/dai/unused_5A70C.c

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "dai.h"
+
+STATIC_PAD_BSS(0xC00);
```

Comment:
Spot checking a few other overlays, it looks like this always follows `create_entity` and is probably part of it. The only exception is `no3` where the `0xC00` padding is in defined `cutscene.c`.

I think it makes sense to leave this out and someone can go back and put this in `create_entity` for all of the stages.